### PR TITLE
Navigation

### PR DESCRIPTION
**Pull request by Ananyamadhu08**
- created top nav bar
- created collapsible top bar
- created side nav bar
Screen Shot 2022-02-15 at 4 24 01 AM
Screen Shot 2022-02-15 at 4 24 27 AM
Screen Shot 2022-02-15 at 4 24 34 AM

### CODE REVIEW

File: css/navigation.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  z-index: 10;
+  width: 100%;
+  padding: 0 1.5rem;
+  min-height: 5rem;
+  display: flex;
+}
+
+.top-nav .links {
+  float: right;
+  display: flex;
```
Comment (ShrutiLakhani) [inline css/navigation.css:14]: We should avoid using float.

File: css/navigation.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  display: none;
+}
+
+.top-nav .icon-burger {
+  display: none;
+  transform: translate(-50%);
+}
+
+.top-nav .icon-burger .line {
+  width: 30px;
```
Comment (ShrutiLakhani) [inline css/navigation.css:27]: Avoid using px for anything over 10px

File: css/navigation.css [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    position: fixed;
+    left: 0;
+    right: 0;
+    top: 5rem;
+    bottom: 100%;
+    height: auto;
+    flex-direction: column;
+    justify-content: space-evenly;
+    align-items: center;
+    background-color: black;
```
Comment (ShrutiLakhani) [inline css/navigation.css:45]: Please use color variables

File: css/navigation.css [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  }
+
+  .top-nav :checked ~ .icon-burger .line:nth-child(2) {
+    opacity: 0;
+  }
+}
+
+/* fixed side nav */
+.fixed-side-nav {
+  background-color: firebrick;
```
Comment (ShrutiLakhani) [inline css/navigation.css:76]: Same here .. color variables

**Review by ShrutiLakhani — CHANGES_REQUESTED** [review]
Changes suggested

**Review by SinchanaMS — COMMENTED** [review]
Neat code, Ananya! 
Use color variables and rem units for over 10px as already suggested. The rest is good to merge.

**Pull request merged**